tests/10-EdvClient.spec.js

```diff
@@ -1,6 +1,7 @@
 /*!
  * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
  */
+
```

Comment:
Remove unnecessary whitespace. Same thing in a number of these files.

Reply:
that means the import statements should be like this:

```js
/*!
 * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
 */
import crypto from '../crypto.js';
```

That is fine with me, it's just I usually add a line between copyright and import statements.

Reply:
I believe this is addressed here: https://github.com/digitalbazaar/edv-client/pull/26/commits/2f089e25f223ae2486c82898036d964a2153686c